### DISCUSSION TITLE
Firefox Trash

### DISCUSSION THREAD

**Discussion by SavSanta** [mozilla-mobile/focus-android#7938] [Q&A] [answered]
Hi,

Unfortunately I chose to be dumb and randomly update the Android app "Firefox Focus". Not testing out which dumb design changes had been made like I normally do as I know better from previous bad UI changes to try before the leap.

I see now that the Trash can to delete tabs  moved to be precariously placed right next to the side of the url bar. So if it is hit accidentally there goes all your tabs with no confirmation or anything.

What is the last version that had a properly functioning Trash can. Where it was the floating button that had the safety aspect of a psuedo two-step process before wiping all your tabs? And wasn't as accident prone as this new version?

Regards

**Comment by Mugurell** [comment, marked as answer]
According to https://github.com/mozilla-mobile/focus-android/issues/5829 the functionality was relocated almost a year ago, starting with version 96.
Using such an old version is not recommended given that in this time Firefox Focus got many new improvements, performance updates, security and privacy features.
As described in the above ticket moving the button helped fix other scenarios and is consistent with the UX from other browsers.

**Reply by SavSanta** [reply]
It sounds like around 95.0 series sounds is correct. A site i used that has User-Agent history already rotated so i cant verify. Tho ill try this later. As far as purported security and performance improvement aspects. My threat model on the phone is already low and barriers in place. Performance was fine and only one or two bugs. So Ill have to forgo the improvements until the long-term where it is fixed.  
This is Answered.

Cheers.

**Comment by Mugurell** [comment]
Opened https://github.com/mozilla-mobile/focus-android/issues/7945 to investigate possible ways to avoid unintentionally closing tabs.
